Adding to faqs (#11)
tenstorrent/tt-buda-demos: FAQ.md

## Resetting an accelerator board

If you encounter a situation where a Tenstorrent chip appears to be unresponsive or is exhibiting unusual behavior, a software reset of the board might be a viable solution.

For a software reset on a single chip use : `tt-smi -lr 0` .

> For more information on performing reset on multiple chips or other specifics visit [TT-SMI homepage](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)

Reviewer: ```suggestion
For more information on performing reset on multiple chips or other specifics visit [TT-SMI Resets](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)
```

If you need additional assistance, you can access a detailed explanation of all available command options by appending the help flag to the command like so: `tt-smi --help` or `tt-smi -h`.

> For comprehensive insights and detailed instructions on utilizing the command line GUI, we invite you to explore the Tenstorrent System Management Interface (TT-SMI) repository on GitHub at [tt-smi-repo](https://github.com/tenstorrent/tt-smi). TT-SMI serves as a versatile command-line utility tailored to streamline interaction with all Tenstorrent devices on host.

Reviewer: Nit-picking the links.

```suggestion
For comprehensive insights and detailed instructions on utilizing the command line GUI, we invite you to explore the Tenstorrent System Management Interface (TT-SMI) repository on GitHub at [TT-SMI homepage](https://github.com/tenstorrent/tt-smi). TT-SMI serves as a versatile command-line utility tailored to streamline interaction with all Tenstorrent devices on host.
```

## Resetting an accelerator board

If you encounter a situation where a Tenstorrent chip appears to be unresponsive or is exhibiting unusual behavior, a software reset of the board might be a viable solution.

> For Grayskull: `tt-smi -tr all`

Reviewer: Remove this line. It's redundant if you are mentioning `tt-smi -lr 0`.

Author: removing

For a software reset on a single chip use : `tt-smi -lr 0` .

> For more information on performing reset on multiple chips or other specifics visit this [link](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)

Reviewer: Specify where the link leads to

```suggestion
For more information on performing reset on multiple chips or other specifics visit [TT-SMI homepage](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)
```
